docs/resources/Channel.md

```diff
@@ -1190,24 +1192,28 @@ When called on a `GUILD_TEXT` channel, creates a `GUILD_PUBLIC_THREAD`. When cal
 
 ###### JSON Params
 
-| Field                 | Type    | Description                                                                                                         |
-|-----------------------|---------|---------------------------------------------------------------------------------------------------------------------|
-| name                  | string  | 2-100 character channel name                                                                                        |
-| auto_archive_duration | integer | duration in minutes to automatically archive the thread after recent activity, can be set to: 60, 1440, 4320, 10080 |
+| Field                   | Type    | Description                                                                                                         |
+|-------------------------|---------|---------------------------------------------------------------------------------------------------------------------|
+| name                    | string  | 2-100 character channel name                                                                                        |
+| auto_archive_duration\* | integer | duration in minutes to automatically archive the thread after recent activity, can be set to: 60, 1440, 4320, 10080 |
+
+\* The 3 day and 7 day archive durations require the server to be boosted. The [guild features](#DOCS_RESOURCES_GUILD/guild-object-guild-features) will indicate if a server is able to use those settings.
 
 ## Start Thread without Message % POST /channels/{channel.id#DOCS_RESOURCES_CHANNEL/channel-object}/threads
 
-Creates a new thread that is not connected to an existing message. The created thread is always a `GUILD_PRIVATE_THREAD`. Returns a [channel](#DOCS_RESOURCES_CHANNEL/channel-object) on success, and a 400 BAD REQUEST on invalid parameters. Fires a [Thread Create](#DOCS_TOPICS_GATEWAY/thread-create) Gateway event.
+Creates a new thread that is not connected to an existing message. The created thread is always a `GUILD_PRIVATE_THREAD`\*. Returns a [channel](#DOCS_RESOURCES_CHANNEL/channel-object) on success, and a 400 BAD REQUEST on invalid parameters. Fires a [Thread Create](#DOCS_TOPICS_GATEWAY/thread-create) Gateway event.
```

Comment:
> The created thread is always a `GUILD_PRIVATE_THREAD`
How would this work if the guild doesn't have the ability to use private threads? This default, as mentioned further down, was kept to match the behavior from when the docs were first published but now that private threads are (unfortunately) locked behind boosts, most servers won't be able to use them, making keeping this default kinda useless from what I can see

Reply:
The request will fail with a 400
I think that's the right behavior, because the alternative is a bot that didn't get the memo so it thinks it is about to create a private thread and then dun dun dun turns out now it's a public thread. Surprise!

Reply:
This should actually say `defaults to` now anyways since it isn't *always* a private thread anymore